test/check-machines-settings

```diff
@@ -406,6 +406,7 @@ class TestMachinesSettings(VirtualMachinesCase):
 
         # Change vCPUs setting
         b.click("#vm-subVmTest1-vcpus-count button")  # Open dialog
+        b.wait_visible("#machines-vcpu-modal-dialog")
```

Comment:
How does that fix a race? It's a  good check on its own (we do expect the dialog), but the following set_input_text() will wait for the vcpu-max-field to appear anyway, so it shouldn't make much difference. Also, the test in this PR fail on the tooltip